packages/cli/src/test-utils/setup.global.ts

```diff
@@ -1,6 +1,5 @@
 import baseSetup from '@celo/dev-utils/lib/ganache-setup'
 // Has to import the matchers somewhere so that typescript knows the matchers have been made available
-import _unused from '@celo/dev-utils/lib/matchers'
```

Comment:
gonna have a bad time if this is removed.

Reply:
damn it everytime, my editor removes it automagically